eth/handler_eth.go

```diff
@@ -70,6 +70,9 @@ func (h *ethHandler) Handle(peer *eth.Peer, packet eth.Packet) error {
 	case *eth.NewPooledTransactionHashesPacket:
 		return h.txFetcher.Notify(peer.ID(), *packet)
 
+	case *eth.NewPooledTransactionHashesPacket68:
```

Comment:
Nitpick, do this the other way around, tag 67. The rationale being that that will be the version we'll drop and it's easier to see what to drop vs. having to rename the 68 afterwards.

Reply:
No need to tag the latest version :P That was the goal :)